src/App.js

```diff
@@ -0,0 +1,80 @@
+import { useState } from "react";
+
+function Square({ value, onSquareClick }) {
+  return (
+    <button className="square" onClick={onSquareClick}>
+      {value}
+    </button>
+  );
+}
+
+export default function Board() {
+  const [xIsNext, setXIsNext] = useState(true);
```

Comment:
what is xIsNext? Naming looks unclear to me
```suggestion
  const [xIsNext, setXIsNext] = useState(true);
```